module_generator/cookiecutter-template/cookiecutter.json

```diff
@@ -0,0 +1,76 @@
+{
+  "module_name": "MyModule",
+  "language": ["python"],
+  "namespace": "ns",
+  "resource": [
```

Comment:
maybe more for a v2, but I think when we're looking at all these resource types we end up with a potentially overwhelming list in the user interface. I wonder if some sort of fuzzy finding support is possible?

Reply:
I personally had no issue with this, as it gives a list of the components/services that are available. Although you're right that this can get messy easily if the list gets bigger than what it is now, or maybe it already is too big. Right now, it looks like this:
<img width="710" alt="image" src="https://github.com/user-attachments/assets/c8e113a2-b6c1-472b-a325-4757f160e35f">

(Had to look up what fuzzy search was) If we want to implement a fuzzy search, I do agree that should be a v2.